dev-docs/source/MVPTutorial/Layouts.rst

```diff
@@ -8,32 +8,32 @@ next to the button. This is possible by using **layouts**.
 
 So far we have used the vertical layout (``QtWidgets.QVBoxLayout``) and we
```

Comment:
I think up to this point the only used layout was a GridLayout. 
Is it possible to either change the layout of the previous examples to QVBox or specify here that so far we have used the Grid Layout?